newIDE/app/src/EventsSheet/InlineParameterEditor.js

```diff
@@ -123,6 +123,7 @@ export default class InlineParameterEditor extends React.Component<
           parameterIndex={this.props.parameterIndex}
           value={instruction.getParameter(this.props.parameterIndex)}
           onChange={this.props.onChange}
+          onRequestClose={this.props.onRequestClose}
```

Comment:
We probably want to remove this if not useful for now?

Reply:
Oh no, not this one, I guess this is passed for ClickAwayListener? We need atleast that to close the inline popover, no?

Reply:
I don't know, I mean this is something added in this PR, and it already works in the current version no?
I don't see a `ClickAwayListener`, there is one in the InlinePopover if I recall properly, but this is not the responsibility of the ParameterComponent to deal with it, right?

Reply:
As far as I remember it was passed to the ClickAwayListener by the InlineParameterEditor before I started working on this. (Also this is how I thought of passing it even down, to the Autocomplete component. Maybe too much commits, adding and removing this polluted the history? )

Reply:
Not sure, git is pretty robust :) Check if you can remove this. It's being added in this PR and I'm not sure it's useful